kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml

```diff
@@ -1779,7 +1787,7 @@
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>findbugs-maven-plugin</artifactId>
           <configuration>
-            <excludeFilterFile>${project.basedir}/src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
+            <excludeFilterFile>${project.basedir}/../../src/main/findbugs/findbugs-exclude.xml</excludeFilterFile>
```

Comment:
@manstis it goes back to the root folder so that the same findbugs configuration is shared across kie-wb and kie-drools-wb.